command.py

```diff
@@ -0,0 +1,52 @@
+from .plugin import LspIntelephensePlugin
+from LSP.plugin.core.typing import Optional, TypeVar
+import os
+import shutil
+import sublime
+import sublime_plugin
+
+T = TypeVar("T")
+
+
+def expand_variables(var: T, window: Optional[sublime.Window] = None) -> T:
+    if not window:
+        window = sublime.active_window()
+
+    variables = window.extract_variables()
+    variables.update(LspIntelephensePlugin.additional_variables() or {})
+
+    return sublime.expand_variables(var, variables)
+
+
+class LspIntelephenseClearCacheCommand(sublime_plugin.WindowCommand):
+    """ Clear the cache generated by intelephense. """
+
+    def run(self) -> None:
+        settings = sublime.load_settings("LSP-intelephense.sublime-settings")
+
+        cache_path = settings.get("initializationOptions", {}).get("storagePath", "")  # type: str
+        cache_path = os.path.realpath(os.path.abspath(expand_variables(cache_path)))
```

Comment:
It would be much better if you could get the settings from LSP itself. Wouldn't have to expand and would give you support for overridden settings in the project, for example.

Reply:
> It would be much better if you could get the settings from LSP itself.

But how?